internal/routing/local.go

```diff
@@ -41,7 +41,7 @@ func (r *Routing) LocalNetworks() (localNetworks []LocalNetwork, err error) {
 		return localNetworks, fmt.Errorf("%w: in %d links", ErrLinkLocalNotFound, len(links))
 	}
 
-	routes, err := r.netLinker.RouteList(nil, netlink.FAMILY_V4)
+	routes, err := r.netLinker.RouteList(nil, netlink.FAMILY_ALL)
```

Comment:
Is there a way to only list v4 and v6? 🤔 
Or maybe add another route list and merge the slices together? I'm just worried it would list other routes we don't care about (if it exists).

Reply:
As far as I know, there's only V6 and V4 existing.